apps/docs/src/app/core/component-docs/object-status/examples/object-status-clickable-and-icon-example.component.html

```diff
@@ -1,10 +1,10 @@
-<a fd-object-status [status]="'negative'" [glyph]="'status-negative'" [clickable]="true">Negative</a>
-<a fd-object-status [status]="'critical'" [glyph]="'status-critical'" [clickable]="true">Critical</a>
-<a fd-object-status [status]="'positive'" [glyph]="'status-positive'" [clickable]="true">Positive</a>
-<a fd-object-status [status]="'informative'" [glyph]="'hint'" [clickable]="true">Informative</a>
-<a fd-object-status [glyph]="'to-be-reviewed'" [clickable]="true">Default</a>
+<a fd-object-status status="negative" glyph="status-negative" clickable="true" label="Negative"></a>
+<a fd-object-status status="critical" glyph="status-critical" clickable="true" label="Critical"></a>
+<a fd-object-status status="positive" glyph="status-positive" clickable="true" label="Positive"></a>
+<a fd-object-status status="informative" glyph="hint" clickable="true" label="Informative"></a>
+<a fd-object-status glyph="to-be-reviewed" clickable="true" label="Default"></a>
```

Comment:
Boolean values can't have static binding (false would be treated as true won't be applied in this case)